spec/hbw/features/select_table_spec.rb

```diff
@@ -111,25 +185,14 @@
       expect_widget_presence
     end
 
-    scenario 'multi = false' do
```

Comment:
Why did you remove it?

Reply:
Because functionality of select table is changed and this case is not actual